packages/cashier/src/types/stores/root-store.types.ts

```diff
@@ -0,0 +1,16 @@
+import { TClientStore } from './client-store.types';
+import { TCommonStore } from './common-store.types';
+import { TUiStore } from './ui-store.types';
+
+export class RootStore {
```

Comment:
@george-usynin-binary Just a question, Is there any reason we use a class here instead of `type` or `interface`? 🤔 

If not then I suggest changing it to a `type` to keep it consistent with all other types we have.

Reply:
@farzin-fs sure, we can change it into `type`, thanks for the notice